src/Form/Constraint/VimeoValidator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Message\Cog\Form\Constraint;
```

Comment:
Yeah, I flip-flipped between the two. The thing is, Mothership consistently uses singulars. So does Symfony for the most part, except for the `Constraints` namespace. I don't really want to inherit something that I think is incorrectly named in Symfony into Mothership.